hbase-server/src/test/java/org/apache/hadoop/hbase/master/TestRecreateCluster.java

```diff
@@ -0,0 +1,185 @@
+/**
```

Comment:
nit: this should be a non-javadoc comment

Reply:
fixed, sorry that I copied from other files and i found many place in hbase has this header lol